### PR TITLE
Init: Integration of air pollutants like  NOx and PM10 for a car

### PR DESCRIPTION
**Pull request by kadhikari**
For details: https://navitia.atlassian.net/browse/NAV-1760

### CODE REVIEW

File: source/jormungandr/jormungandr/scenarios/helpers.py [hunk 1]
```diff
@@ -30,6 +30,10 @@
 from __future__ import absolute_import, print_function, unicode_literals, division
 from navitiacommon import response_pb2
 
+AIR_POLLUTANTS_ESTIMATION_NOX_COEFF = 1.35
+AIR_POLLUTANTS_ESTIMATION_PM10_COEFF = 184
```
Comment (SGrenet) [inline source/jormungandr/jormungandr/scenarios/helpers.py]: values choice have been done by PO

**Review by SGrenet — APPROVED** [review]

File: source/jormungandr/jormungandr/scenarios/helpers.py (hunk 1 above)
Comment (kadhikari) [reply source/jormungandr/jormungandr/scenarios/helpers.py]: These are fictive values for init

**Review by pbench — APPROVED** [review]

File: source/jormungandr/jormungandr/scenarios/helpers.py [hunk 2]
```diff
@@ -30,6 +30,10 @@
 from __future__ import absolute_import, print_function, unicode_literals, division
 from navitiacommon import response_pb2
 
+AIR_POLLUTANTS_ESTIMATION_NOX_COEFF = 0.44
+AIR_POLLUTANTS_ESTIMATION_PM10_COEFF = 0.056
```
Comment (woshilapin) [inline source/jormungandr/jormungandr/scenarios/helpers.py]: Can we put comments about where these values come from?

**Review by woshilapin — APPROVED** [review]

File: source/jormungandr/jormungandr/scenarios/helpers.py (hunk 2 above)
Comment (kadhikari) [reply source/jormungandr/jormungandr/scenarios/helpers.py]: There is no official resource. I took it from the ticket as proposed.

Comment (woshilapin) [reply source/jormungandr/jormungandr/scenarios/helpers.py]: There are some details in https://navitia.atlassian.net/browse/NAV-1760?focusedCommentId=39713. We could make a link to official documents and explain how we kept this average value. Maybe @wassimbenaissa can help to do that?

Comment (kadhikari) [reply source/jormungandr/jormungandr/scenarios/helpers.py]: Done

**Review by pbench — APPROVED** [review]

**Pull request merged**